cmd/keytransparency-signer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang
+
+ENV DB_USER=test \
+    DB_PASSWORD=zaphod \
+    DB_DATABASE=test \
+    DB_HOST=127.0.0.0:3306
+
+ENV MAP_ID=0 \
+    MAP_URL=""
+ENV LOG_ID=0 \
+    LOG_URL=localhost:8090 \
+    LOG_KEY=genfiles/trillian-log-pubkey.pem
+
+ENV SIGN_PERIOD=5s \
+    SIGN_KEY=genfiles/p256-key.pem
+
+ADD . /go/src/github.com/google/keytransparency
+WORKDIR /go/src/github.com/google/keytransparency 
+
+RUN go get -tags="mysql" ./cmd/keytransparency-signer
+
+ENTRYPOINT /go/bin/keytransparency-signer \
+	--db="${DB_USER}:${DB_PASSWORD}@tcp(${DB_HOST})/${DB_DATABASE}" \
+	--period="$SIGN_PERIOD" --key="$SIGN_KEY" \
```

Comment:
The private signing key `--key="$SIGN_KEY"` is treated as [path to a file flag](https://github.com/google/keytransparency/blob/55401b8b31610398d0966b0dc4a441e11345763e/cmd/keytransparency-signer/backend.go#L54) here and yet the passed string is interpreted to contain the PEM bytes [here](https://github.com/google/keytransparency/blob/55401b8b31610398d0966b0dc4a441e11345763e/cmd/keytransparency-signer/backend.go#L84). Looks like this fails (but independent from this docker pull).